### DISCUSSION TITLE
Getting Picklist items from the database

### DISCUSSION THREAD

**Discussion by StephenFeddes** [YetiForceCompany/YetiForceCRM#17151] [How to]
Hi, I am making an application that will read from our yetiforce MySQL database. I need to get all the list items that are in my account's tower_name picklist. How can I do this? Which tables should I be looking at? We have the newest version of Yetiforce.

**Comment by mariuszkrzaczkowski** [comment]
why don't you use api?  there you have it ready

**Reply by StephenFeddes** [reply]
I looked online for the API documentation, but I couldn't find what I need. Would it be easier for you to tell me which tables I should look at, or which endpoints to use? I'm just using a read-only database user.

**Reply by StephenFeddes** [reply]
Is there any documentation on how to use the API? Do I have to create an API user inside Yetiforce itself? I need a step-by-step tutorial.

**Comment by StephenFeddes** [comment]
Where can I find the API documentation? How do I get authenticated?

**Reply by mariuszkrzaczkowski** [reply]
https://doc.yetiforce.com/developer-guides/api/
https://github.com/YetiForceCompany/YetiForceScripts/tree/master/YetiREST
